var/spack/repos/builtin/packages/hpe-mpi/package.py

```diff
@@ -6,12 +33,35 @@ class HpeMpi(Package):
     homepage = "http://www.example.com"
     url      = "http://www.example.com/hpempi-1.0.tar.gz"
 
-    version('2.16', '0123456789abcdef0123456789abcdef')
+    version('2.21', '2dd6c53a82993c4df929fdf898ac3d19',
+        url='file:///gpfs/bbp.cscs.ch/apps/hpc/download/hpe-mpi/hpe-mpi-2.21.tar.xz')
```

Comment:
can you sync with Dries to see how these can be uploaded to pulp server https://bbpteam.epfl.ch/repository/pulp/ ?

Reply:
OK, I'll check.

Reply:
Let's stick with this URI for now